sormas-backend/src/main/java/de/symeda/sormas/backend/caze/surveillancereport/SurveillanceReportService.java

```diff
@@ -50,24 +53,25 @@ public Predicate buildCriteriaFilter(SurveillanceReportCriteria criteria, Criter
 		return filter;
 	}
 
-	public List<SurveillanceReport> getByCaseUuid(String caseUuid) {
-		CriteriaBuilder cb = em.getCriteriaBuilder();
-		CriteriaQuery<SurveillanceReport> cq = cb.createQuery(SurveillanceReport.class);
-		Root<SurveillanceReport> sampleRoot = cq.from(SurveillanceReport.class);
-		Join<SurveillanceReport, Case> caseJoin = sampleRoot.join(SurveillanceReport.CAZE, JoinType.LEFT);
+	public List<SurveillanceReport> getByCaseUuids(List<String> caseUuids) {
 
-		cq.where(caseJoin.get(AbstractDomainObject.UUID).in(caseUuid));
-		return em.createQuery(cq).getResultList();
-	}
+		if (caseUuids != null) {
```

Comment:
There could still be an error when calling this with an empty caseUuids list because I think Hibernate can't deal with empty lists in IN clauses